### PR TITLE
Parse crawlspaces

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds additional functionality to `basements` to also parse crawlspace data.

### CODE REVIEW

**Comment by buckley-w-david** [comment]
Just noticed something that was missed in parsing, changing to [WIP]

File: etl/pylintrc [hunk 1]
```diff
@@ -208,10 +208,10 @@ attr-rgx=[a-z_][a-z0-9_]{2,30}$
 attr-name-hint=[a-z_][a-z0-9_]{2,30}$
 
 # Regular expression matching correct argument names
-argument-rgx=[a-z_][a-z0-9_]{2,30}$
+argument-rgx=[a-z_][a-z0-9_]{2,120}$
 
 # Naming hint for argument names
-argument-name-hint=[a-z_][a-z0-9_]{2,30}$
+argument-name-hint=[a-z_][a-z0-9_]{2,120}$
```
Comment (sastels) [inline etl/pylintrc]: Why 120?

Comment (buckley-w-david) [reply etl/pylintrc]: That's the max line length we have set.
I had to change it because one of the test fixtures names in the PR is 32 characters

**Comment by buckley-w-david** [comment]
[Extract/Load Foundation data](https://trello.com/c/MHCxw57a/107-extract-load-foundation-data)

**Pull request merged**